Readme.md

```diff
@@ -54,6 +42,49 @@ t.integer :my_bits, default: 0, null: false
 add_column :users, :my_bits, :integer, default: 0, null: false
 ```
 
+Instance Methods
```

Comment:
These tables are a bit crowded / repetitive, maybe only use 2 columns and put the example up top or so 🤷‍♂